### ISSUE TITLE
PWAs are being groups with Chrome browser icons in gnome

### ISSUE THREAD

**Issue by anthr76** [flathub/com.google.Chrome#205]
I've detailed this a bit here https://discussion.fedoraproject.org/t/chrome-pwas-in-gnome-grouping-into-the-same-browser-icon/81714

I've been following the paths between Flextop and Cobalt trying to find a root cause to this issue.

It seems at one point in the past this used to work, but now it doesn't? A fresh PWA installed as `com.google.Chrome.flextop.chrome-cifhbcnohmdccbgoicgdjpfamggdegmo-Profile_1.desktop` which seems to have the correct prefix necessary for to have a specific icon and window to not be grouped. 

```
cat com.google.Chrome.flextop.chrome-cifhbcnohmdccbgoicgdjpfamggdegmo-Profile_1.desktop | grep -i Start
StartupWMClass=crx_cifhbcnohmdccbgoicgdjpfamggdegm
```

**Comment by chrisawi** [comment]
Have you configured Chrome to use Wayland? If so, this is the same as https://github.com/flathub/org.chromium.Chromium/issues/216.

I also just filed https://github.com/refi64/flextop/issues/9 against the underlying component responsible.

For this specific example, the workaround would be to edit the .desktop file and change StartupWMClass  to `chrome-cifhbcnohmdccbgoicgdjpfamggdegmo-Profile_1`

**Comment by anthr76** [comment]
@chrisawi That's correct. Thanks! Should we close this issue and move over to your filed issue
